.github/workflows/chrometest_run-batches.yml

```diff
@@ -142,6 +166,27 @@ jobs:
           .\nunit3-console.exe ChromeTests.dll --encoding=utf-8 --domain=None --inprocess --labels=All --testparam non-interactive=true `
             --result=${{ env.RERUN_RESULT_FILE }} `
             --testlist $OriginalFailTestResult
+      # Assuming average of 30 chrometests per day, total size of text files for a run should be ~300 kb, these will expire after 7 days, so 7x30x300 = 63 MB used at any time
```

Comment:
Can remove this, basically just repeating the rerun step on job reruns. Need to cache the result file to do so.

Reply:
We can do this a bit later.

Lets just make sure the chrometest are stable first. Cause we are making a big change atm